src/components/InventoryV2/view/InventorySeedlingsTableForNursery.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useCallback } from 'react';
+import { TableColumnType } from '@terraware/web-components';
+import strings from 'src/strings';
+import { FieldNodePayload, SearchResponseElement, SearchSortOrder } from 'src/types/Search';
+import { NurseryBatchService } from 'src/services';
+import InventorySeedlingsTable, { InventorySeedlingsTableProps } from './InventorySeedlingsTable';
+
+interface InventorySeedlingsTableForNurseryProps
+  extends Omit<
+    InventorySeedlingsTableProps,
+    'columns' | 'isSelectionBulkWithdrawable' | 'getFuzzySearchFields' | 'getBatchesSearch' | 'getBatchesExport'
+  > {
+  nurseryId: number;
+}
+
+const columns: TableColumnType[] = [
+  { key: 'batchNumber', name: strings.SEEDLING_BATCH, type: 'string' },
+  { key: 'species_scientificName', name: strings.SPECIES, type: 'string' },
+  { key: 'germinatingQuantity', name: strings.GERMINATING, type: 'string' },
+  { key: 'readyQuantity', name: strings.READY, type: 'string' },
+  { key: 'notReadyQuantity', name: strings.NOT_READY, type: 'string' },
+  { key: 'totalQuantity', name: strings.TOTAL, type: 'string' },
+  { key: 'totalQuantityWithdrawn', name: strings.WITHDRAWN, type: 'string' },
+  { key: 'readyByDate', name: strings.EST_READY_DATE, type: 'date' },
+  { key: 'addedDate', name: strings.DATE_ADDED, type: 'date' },
+  { key: 'withdraw', name: '', type: 'string' },
+  { key: 'quantitiesMenu', name: '', type: 'string' },
+];
+
+export default function InventorySeedlingsTableForNursery(props: InventorySeedlingsTableForNurseryProps): JSX.Element {
+  const facilityId = props.nurseryId;
```

Comment:
This is done with the intention that we might use the `InventorySeedlingsTable` for other facility types in the future.